apps/core-app/src/main/utils/app-root-path.ts

```
@@ -0,0 +1,200 @@
+  error?: string
+}
+
+function normalizePath(input: string): string {
+  const normalized = path.resolve(input)
+  return process.platform === 'win32' ? normalized.toLowerCase() : normalized
+}
+
+function safeGetUserDataPath(appLike: AppPathLike, fallbackBasePath: string): string {
+  try {
+    return appLike.getPath('userData')
+  } catch {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)